include/SFML/Graphics/Image.hpp

```diff
@@ -75,7 +75,7 @@ class SFML_GRAPHICS_API Image
     /// \param pixels Array of pixels to copy to the image
     ///
     ////////////////////////////////////////////////////////////
-    void create(const Vector2u& size, const std::uint8_t* pixels);
+    Image(const Vector2u& size, const std::uint8_t* pixels);
```

Comment:
```suggestion
    explicit Image(const Vector2u& size, const std::uint8_t* pixels);
```

`explicit` is a good default even for multiple-argument constructors as it still forces `Image` to be spelled out when constructing from a braced initialization list.

Reply:
I'm personally not a fan of forcing the use of the type name for multi-param constructors. SFML conventionally does not do this so I'd rather not start doing so now. If we want to revisit our use of `explicit` then I want that to happen in a separate PR.